### ISSUE TITLE
Incorrect report of comma sequence

### ISSUE THREAD

**Issue by Anahkiasen** [brigade/scss-lint#190]
I have the following interpolated selector:

``` scss
label.inline,
#{$value-inputs},
select {
    margin-bottom: 0;
}
```

`$value-inputs` is itself a list of selectors. scss-linter is throwing the following error but I can't seem to understand why:

```
public/app/sass/modules/_forms.scss:228 [W] Each selector in a comma sequence should be on its own line
```

If I replace the variable interpolation with a classic element (like `textarea`) it works. Here's the variable's declaration:

``` scss
$value-inputs:
    'input[type="text"],
    input[type="password"],
    input[type="date"],
    input[type="datetime"],
    input[type="datetime-local"],
    input[type="month"],
    input[type="week"],
    input[type="email"],
    input[type="number"],
    input[type="search"],
    input[type="tel"],
    input[type="time"],
    input[type="url"],
    textarea';
```
